gdal/ogr/ogr_srs_panorama.cpp

```diff
@@ -150,6 +150,44 @@ constexpr int aoEllips[] =
 
 constexpr int NUMBER_OF_ELLIPSOIDS = static_cast<int>(CPL_ARRAYSIZE(aoEllips));
 
+/************************************************************************/
+/*  Correspondence between "Panorama" and EPSG vertical CS.             */
+/************************************************************************/
+
+constexpr long aoVCS[] =
```

Comment:
we could probably just use ``int`` everywhere ``long`` is used

Reply:
No problem.

Here I tried to make minimal changes compared to ogrsxfdatasource.